test/util/bitcoin-util-test.py

```diff
@@ -43,12 +43,11 @@ def main():
     formatter = '%(asctime)s - %(levelname)s - %(message)s'
     # Add the format/level to the logger
     logging.basicConfig(format=formatter, level=level)
-
```

Comment:
nit: why remove this line? Blank lines are used within functions to indicate logical sections.

Reply:
sorry, that was mistake while rebasing, I'll add again now

Reply:
You've now added trailing whitespace :(